setup.py

```diff
@@ -6,15 +6,15 @@
 
 requirements = [
     "funcy==1.18",
-    "pefile==2022.5.30",
-    "vivisect==1.0.8",
+    "pefile>=2023.2.7",
```

Comment:
i think i've learned my lesson here: for libraries to be used by applications, use >= so that they can be updated downstream